tests/nested/manual/preseed/task.yaml

```diff
@@ -41,6 +41,13 @@ prepare: |
   mv snapd-from-deb.snap snapd.snap
   inject_snap_into_seed "$IMAGE_MOUNTPOINT" snapd
 
+  # inject a snap that uses system-usernames into the seed to confirm that works
+  # as expected
+  # TODO: replace this snap with a simpler one instead that is smaller, this one
+  # is 37M, but test-snapd-daemon-user does not have a daemon yet
+  snap download --edge --basename=test-postgres-system-usernames test-postgres-system-usernames
+  inject_snap_into_seed "$IMAGE_MOUNTPOINT" test-postgres-system-usernames
```

Comment:
Nice! 
Is there any sensible check we could do below under "Checking that lxd snap is operational" to verify that that snap really used system-usernames? Could we grep snap.yaml and verify the daemon was started succesfuly like we do with lxd below?

Reply:
yeah I'll push something I had that in my original patch I think but I lost that with all the force-pushes to #9041 so I'll try to add something here

Reply:
done